.github/workflows/pull_request.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
         uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
         with:
           version: latest
+          args: -v -c .golangci.yaml
```

Comment:
I believe we weren't picking up our local `.golangci.yaml` config because by default this picks up `.golangci.yml` (notice how the extension is yml instead of yaml) as mentioned in https://github.com/golangci/golangci-lint-action#how-to-use.